
---
int32 dirAntMaxAngle
int32 dirAntSweepNumber
int32 dirAntLevel

